### DISCUSSION TITLE
Is it possible to have one field have a different validation method?

### DISCUSSION THREAD

**Discussion by JopV** [react-hook-form/react-hook-form#8685] [Q&A] [answered]
We would like our form to have the validation method `all`, with the exception of our email field, where we would like it to have the method `onBlur`. However, this is not an option with useController props.
Is there a way this can be achieved within the same form?

**Comment by bluebill1049** [comment, marked as answer]
have a read on this thread: https://github.com/react-hook-form/react-hook-form/issues/8649

**Reply by JopV** [reply]
Thank you.
I went with `onChange={disableChangeValidation ? e => setValue(name, e.target.value) : onChange}` based on a suggestion in that thread.
